src/app/routes/connector-ui/dashboard-page/dashboard-page/dashboard-page.component.html

```diff
@@ -89,6 +89,24 @@
           <mat-icon>content_copy</mat-icon>
         </button>
       </mat-form-field>
+      <mat-form-field class="w-full" style="margin-bottom: -10px">
+        <mat-label>Connector Management Endpoint</mat-label>
```

Comment:
Management API URL, let's keep things consistend and let's not add a 2nd "endpoint". Feel free to take the AP as source-of-truth